library/jruby

```diff
@@ -1,5 +1,5 @@
-Maintainers: Brian Goff <cpuguy83@gmail.com> (@cpuguy83)
-GitRepo: https://github.com/cpuguy83/docker-jruby.git
+Maintainers: JRuby Admin <admin@jruby.org> (@jruby)
```

Comment:
This is used for when we need to find the "humans" who are responsible for this image (to inform you of major changes that might cause issues, etc), and "@jruby" isn't something we can usefully mention on an issue/PR, so I'd appreciate if we could instead or also list both @headius and @enebo (if email is the hesitation, you can skip that -- the parser will accept `Name (@github)` just as well as `Name <email> (@github)`).

Reply:
Ok, for whatever reason I thought the @jruby referred to Twitter, but if it refers to Github that makes sense. There is no JRuby user.

I can list also @headius and @enebo in this, but we would prefer the emails go to admin@jruby.org for general support. Should we just remove the @github part from the `JRuby Admin` entry?

Reply:
I updated the commit to include @headius and @enebo contact information.

It seems that @jruby can stay based on the contact info for Amazon Corretto, which also includes a "team" entry pointing at the @corretto organization, yes?

https://github.com/docker-library/official-images/blob/master/library/amazoncorretto

Reply:
D'oh... fixed.